src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -37,23 +57,10 @@ class TetheredSelect extends Select {
     const selectWidth = this.wrapper ? this.wrapper.offsetWidth : null;
 
     return (
-      <TetherComponent
-        renderElementTo="body"
-        ref="tethered-component"
```

Comment:
Was this `ref` needed for anything?

Reply:
According to Greg J, no.

Reply:
I believe it was left over from a copy and paste